pkg/rest/endpoints/decoders.go

```diff
@@ -241,6 +261,28 @@ func NewJsonPutDecodeRequestFunc(payloadTypePtr interface{}) gokithttp.DecodeReq
 	}
 }
 
+func NewJsonPatchDecodeRequestFunc() gokithttp.DecodeRequestFunc {
+	jsonDecodeRequestFunc := NewMimeTypeAwareDecodeRequestFunc(
+		nil,
+		map[string]gokithttp.DecodeRequestFunc{
+			string(api.JSONPatchType):  JsonPatchDecodeRequestFunc,
+			string(api.MergePatchType): JsonPatchDecodeRequestFunc,
+		},
+	)
+	return func(ctx context.Context, r *http.Request) (interface{}, error) {
+		metadata, err := NameNamespaceDecodeRequestFunc(ctx, r)
```

Comment:
Right. It it says that it extracts a name and a namespace variable from the rest request path.